pluma/schema/outdoor.py

```diff
@@ -25,6 +27,29 @@ def build_schema(root: Union[str, ComplexPath],
     """
     root = ensure_complexpath(root)
     streams = DotMap()
+
+    # Pupil streams
+    streams.PupilLabs.Counter.DecodedFrames =     HarpStream(209, device='PupilLabs', streamlabel='Counter_DecodedFrames', root=root, autoload=autoload, parent_dataset=parent_dataset)
+    streams.PupilLabs.Counter.RawFrames =         HarpStream(210, device='PupilLabs', streamlabel='Counter_RawFrames', root=root, autoload=autoload, parent_dataset=parent_dataset)
+    # N.B. the camera stream has variable length frames in the raw data defined by data bytes in the header. In this schema the raw frame data is not loaded, further processing is needed to extract the full frames.
+    streams.PupilLabs.Data.RawFrames =            PupilStream('PupilLabs/WorldCamera', 
+                                                              [[('SensorId', np.string_, 36)], 
```

Comment:
What does `36` stand for in the `dtype` tuple? Is it a fixed length for the string?

Reply:
Yes, the sensor id string for the pupil sensors is always length 36

Reply:
@RoboDoig cool, in that case would it be easier to simply assume the `dtype` of frame0 and only ask for the structure of the header frame and data frame, maybe as separate named arguments? That would simplify the spec even further.

Reply:
Yes, in this case it would be easier. 

Eventually though we might want a generic NetMQ stream reader, which would need to specify dtypes for all frames.